about/index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="description" content="Домашние задания" />
+    <meta name="keywords" content="front-end" />
+    <meta name="author" content="E.Protasenya" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About me</title>
+  </head>
+  <body>
+    <h1>Обо мне</h1>
+    Пришло время познакомиться! <br />
+    Приветствую всех на моей <del>первой</del> второй страничке, написанной на
+    языке разметки <abbr title="HyperText Markup Language">HTML</abbr>. <br />
+    Меня зовут Евгений, мне 27 лет и путём долгих колебаний, раздумий и
+    глубокого анализа, я отбросил все сомнения и решил кардинально изменить
+    сферу деятельности, записавшись на курс по Front-end разработке в школу
+    <a href="https://intellectfox.by/" target="_blank">Intellectfox</a>.
+    <hr />
+    <blockquote>
+      Я скорее буду делать ошибки, чем ничего. Я скорее испорчу, чем упущу.
+    </blockquote>
+    <hr />
+    <h2>Почему Front-end?</h2>
+    Исследуя сферу IT, изначально front-end разработку даже не рассматривал. Вот
+    он, яркий пример того, как недостаточное изучение вопроса отнимает много
+    сил, времени и мотивации, ведь начал я свой путь с того, что в итоге мне не
+    понравилось от слова совсем. Что же это было останется моим маленьким
+    секретом.
+    <br />
+    Разобравшись, капнув чуть глубже, твердо решил, что должен попробовать свои
+    силы в данном направлении. А помогут мне в этом <ins>усидчивость</ins>,
+    <ins>упорство</ins>, <ins>приступы перфекционизма</ins> и
+    <ins>желание доказать</ins> в первую очередь себе, что я способен на
+    <strong>большее</strong>.
+    <br />
+    <em>Разумеется</em>, свою роль сыграло и желание повысить свой доход.
+    Планирую выйти на заработок в <mark>1000&#8364;</mark> спустя полтора-два
+    года после смены профессии.
+    <br />
+    <h3>Как со мной связаться и где меня найти</h3>
+    Наблюдать за моими успехами можно здесь &#9758;
+    <a href="https://github.com/E-Protasenya" target="_blank">GitHub</a> <br />
+    Написать мне с предложением о работе (ну а вдруг &#x1F60E;) можно сюда
+    &#9758;
+    <a href="mailto:protasenya.evg1994@gmail.com" target="_blank">Mail</a>
+    <br />P.S. К сожалению, как использовать в тексте теги для индекса<sub
+      >2</sub
+    >
+    и степени<sup>2</sup>, я так и не придумал &#x1F60E;
+  </body>
+  <hr />
+  <h2>Image</h2>
+  <img
+    src="https://cdn.flightsim.to/images/01/xcRZgHNE.png?width=1400&auto_optimize=medium"
+    alt="Old Trafford"
+  />
+  <h2>List</h2>
+  <ol>
+    <li>Знакомство с WEB</li>
+    <ul>
+      <li>Краткая история WEB и что такое интернет.</li>
+      <li>Как устроен браузер и как он получает страницы по сети.</li>
+      <li>Что такое Git?</li>
+      <li>Создание первого репозитория.</li>
+      <li>О файле .gitignore.</li>
+    </ul>
```

Comment:
`<ul>` shouldn't be a child of `<ol>`. But it could be a child of `<li>`.
Read about  [context in which this element can be used](https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element).